code/game/objects/structures/door_assembly.dm

```diff
@@ -165,143 +170,188 @@
 		else
 			user.visible_message("<b>[user]</b> begins securing the airlock assembly to the floor.", \
 								SPAN_NOTICE("You start securing the airlock assembly to the floor."))
 
-		if(do_after(user, 40/W.toolspeed))
-			if(!src) return
-			to_chat(user, "<span class='notice'>You [anchored? "un" : ""]secured the airlock assembly!</span>")
+		if(do_after(user, 40 / W.toolspeed))
+			if(!src)
+				return
+			to_chat(user, SPAN_NOTICE("You [anchored? "un" : ""]secure \the [src]."))
 			anchored = !anchored
 
-	else if(W.iscoil() && state == 0 && anchored)
+	else if(W.iscoil())
+		if(state > STATE_UNWIRED)
+			to_chat(user, SPAN_WARNING("\The [src] has already been wired."))
+			return
+		if(!anchored)
+			to_chat(user, SPAN_WARNING("\The [src] must be anchored before it can be wired."))
+			return
 		var/obj/item/stack/cable_coil/C = W
 		if (C.get_amount() < 1)
-			to_chat(user, "<span class='warning'>You need one length of coil to wire the airlock assembly.</span>")
+			to_chat(user, SPAN_WARNING("You need one length of coil to wire the airlock assembly."))
+			return
+		user.visible_message("<b>[user]</b> starts wiring the airlock assembly.", SPAN_NOTICE("You start wiring the airlock assembly."))
+		if(do_after(user, 40) && state == STATE_UNWIRED && anchored)
+			if(C.use(1))
+				state = STATE_WIRED
+				to_chat(user, SPAN_NOTICE("You wire the airlock."))
+
+	else if(W.iswirecutter())
+		if(state == STATE_UNWIRED)
+			to_chat(user, SPAN_WARNING("\The [src] doesn't have any wires to remove."))
+			return
+		else if(state > STATE_WIRED)
+			to_chat(user, SPAN_WARNING("\The [src]'s wires cannot be reached, take out the electronics first."))
 			return
-		user.visible_message("[user] wires the airlock assembly.", "You start to wire the airlock assembly.")
-		if(do_after(user, 40) && state == 0 && anchored)
-			if (C.use(1))
-				src.state = 1
-				to_chat(user, "<span class='notice'>You wire the airlock.</span>")
 
-	else if(W.iswirecutter() && state == 1 )
 		playsound(src.loc, 'sound/items/wirecutter.ogg', 100, 1)
-		user.visible_message("[user] cuts the wires from the airlock assembly.", "You start to cut the wires from airlock assembly.")
-
-		if(do_after(user, 40/W.toolspeed))
-			if(!src) return
-			to_chat(user, "<span class='notice'>You cut the airlock wires.!</span>")
-			new/obj/item/stack/cable_coil(src.loc, 1)
-			src.state = 0
-
-	else if(istype(W, /obj/item/airlock_electronics) && state == 1)
+		user.visible_message("<b>[user]</b> starts cutting the wires from the airlock assembly.", SPAN_NOTICE("You start cutting the wires from airlock assembly."))
+
+		if(do_after(user, 40 / W.toolspeed))
+			if(!src)
+				return
+			to_chat(user, SPAN_NOTICE("You cut the airlock wires."))
+			new /obj/item/stack/cable_coil(src.loc, 1)
+			state = STATE_UNWIRED
+
+	else if(istype(W, /obj/item/airlock_electronics))
+		if(state == STATE_UNWIRED)
+			to_chat(user, SPAN_WARNING("\The [src] doesn't have any wires, use some cable coil on it first."))
```

Comment:
Again not sure about the wording. Usually we try to avoid mentioning specific tools ("use some _cable coil_") because there could be subtypes and whatnot that are different but work the same.

```suggestion
			to_chat(user, SPAN_WARNING("Before inserting \the [W] \the [src] must be wired."))
```